dll/win32/samsrv/lang/ro-RO.rc

```diff
@@ -1,4 +1,6 @@
-/* Translator: Ștefan Fulea (stefan dot fulea at mail dot com) */
+/* Translator: Ștefan Fulea (stefan dot fulea at mail dot com) 
+ *             Andrei Miloiu (miloiuandrei@gmail.com) 
```

Comment:
You may also add the years of contribution (say, the way you did it [here](https://github.com/reactos/reactos/blob/master/dll/cpl/input/lang/ro-RO.rc#L6)).